### ISSUE TITLE
gif and video

### ISSUE THREAD

**Issue by changho3995** [Ashinch/ReadYou#123]
**1. Environment**

-   Device: [Pixel 6]
-   OS: [Android 12.1]
-   APP: [0.7.2]

**2. Describe the feature request**

(Want you've done some research before now)

It would be nice to be able to view gifs and videos within the app.

**Comment by Ashinch** [comment]
@changho3995 GIF is supported, video will take some time.

**Comment by Ashinch** [comment]
@changho3995 Yes, video is unsupported for now.

**Comment by changho3995** [comment]
I hope that it will be possible in the future (?).

**Comment by Ashinch** [comment]
@changho3995 Yes, I will close this issue when the video feature is finished.

**Comment by changho3995** [comment]
Do you know... a service called nitter?
Through that service, I am making twitter as rss and watching it.
Is it possible to have gifs and videos there?

**Comment by BrignLau** [comment]
Is there any progress on this issue?

**Comment by Plazzmex** [comment]
+1 for this.

But, for now, the problem is that I can't even tell the article has videos embedded, because it loads a blank space where video should be.

Is there a way to temporary at least show a thumbnail saying there's a video here, so I know I need to load the website page to see it?
